### ISSUE TITLE
Win10 install only shows mm (never pixels) even when set in Preferences

### ISSUE THREAD

**Issue by hippiecool** [opentoonz/opentoonz#3579]
I have multiple installations on Ubuntu that correctly show pixels (my default preference).  I use a Win10 PC only for rendering since Ubuntu has a permissions issue with snap ffmpeg and doesn't recognize it in the Preferences. The Win10 Pro AMD Ryzen 5 PRO 2400GE with Radeon Vega 11 Graphics computer has the latest updates / patches and its Region Settings are United States.  I installed a fresh copy v1.4 OT from github.  No matter what my Preferences are set to in OT, (e.g. only pixels) the entire UI is in mm.  Pixels are never displayed in the UI, on the Splash Screen or Positions, Camera Settings, Animation, etc making it an annoyance when using OT cross-platform.  Is there something I am missing or did wrong during the Win10 install? Or is this a known issue? 

![Win10 only in mm](https://user-images.githubusercontent.com/20970451/98137218-c178ad00-1e8f-11eb-9fac-c42edc475123.png)

**Comment by shun-iwasawa** [comment]
Do other options in the Preferences work properly?
If no, please check if your stuff folder is with writable permission.
Especially check if the file `C:\OpenToonz stuff\profiles\layouts\settings.username\preferences.ini` ( replace `username` by your user name ) is editable.

If the other options in the Preferences work, I think it's a new issue and I have no idea for now...

**Comment by manongjohn** [comment]
To follow up with what @shun-iwasawa said, make sure you have full permission to the stuff folder. Check in Properties -> Security tab and make sure you see your login there and has full access.

**Comment by manongjohn** [comment]
Aside: Have you tried using the Linux version of OTX to render? It's a portable version built as an AppImage so just untar and run. It doesn't suffer from the snap security issue that prevents using ffmpeg. You can have both versions if you want. It can import some settings from the installed version.

**Comment by hippiecool** [comment]
@shun-iwasawa : Yes other Preferences seem to work as expected.  The preferences.ini is writable and the first few lines of it are below, clearly showing that pixels should be the default.  
@manongjohn :  Permissions on the stuff folder is FULL CONTROL.  I have even reapplied these permissions with no luck.  See the Splash Screen below after re-applying FULL CONTROL and restarting OT.  
So I think something is wrong.   Send me a link to the OTX for Linux.  I am familiar with how AppImages work. 

![Preferences ini](https://user-images.githubusercontent.com/20970451/98249673-871d1780-1f44-11eb-87b0-2e3693d739bb.png)

![Splash](https://user-images.githubusercontent.com/20970451/98250167-1fb39780-1f45-11eb-82b5-1dbfdb163807.png)

**Comment by manongjohn** [comment]
Looks like you are using the Morevna Edition version of OT which might have bugs in it that the Official version does not have.  That version does have some bugs/issues not seen in the Official version.

I would switch to the Official version and see if the problem still exists.

OTX versions can be downloaded here: https://github.com/manongjohn/OTX/releases

**Comment by manongjohn** [comment]
Note, if this issue is only in the Morevna Edition, which I suspect, then please close this and open a bug report on their site (https://github.com/morevnaproject-org/opentoonz)

**Comment by hippiecool** [comment]
Thanks @manongjohn - didn't think about that.  I switched to Morevna Edition because the Ubuntu graphics was poor with the Official OT.  Not sure why but the Morevna Edition didn't have the same graphics issue.  I will re-try and post my findings or close this ticket. 

Thanks for the responsive reply.

**Comment by hippiecool** [comment]
@manongjohn 
- Official OT (Win10) does not have the mm/pixel issue noted above.  It works as expected.  I will close this ticket and open a bug with Morevna. 

- OTX version (Ubuntu) rendering with ffmpeg worked as expected. For other readers who suffer from this issue I have posted my Preferences and Render screens to show you how to have it configured. 

NOTE: OTX version will not auto-launch the completed .mp4 even with Preferences: General: 'Use Default Viewer for Movie Formats' checked.  I validated this using VLC snap, VLC ubuntu repository version, and Ubuntu Videos standard application set in Ubuntu Settings: Default Applications.  It might be a OTX AppImage permissions issue, I suspect.  But at least I can render without having to transfer to Win10 PC. 

![1  Prefs-ffmpeg](https://user-images.githubusercontent.com/20970451/98270319-c192ae80-1f5c-11eb-89a6-2b05456c7853.png)

![2  Output](https://user-images.githubusercontent.com/20970451/98270320-c22b4500-1f5c-11eb-94a2-a90f5534c5b4.png)

![3  Prefs - General](https://user-images.githubusercontent.com/20970451/98270322-c22b4500-1f5c-11eb-8107-64e095d33f3f.png)
